src/app/archive/archive_lib/diff.ml

```diff
@@ -8,7 +8,7 @@ module Breadcrumb = Transition_frontier.Breadcrumb
 module Transition_frontier = struct
   [%%versioned
   module Stable = struct
-    module V2 = struct
+    module V3 = struct
```

Comment:
I'd prefer if we just modify `V2`, since we haven't released it yet.

Reply:
Removing unneeded versioning for these types